_episodes_rmd/10-supp-addressing-data.Rmd

```diff
@@ -147,6 +149,13 @@ We usually use the `$` operator to address a column by name
 dat$Gender
 ```
 
+When we address a data frame by a single column, R will return a vector of that column class and not a data frame.
```

Comment:
I don't think "address a data frame by a single column" is a good description of the `$` operation. Please rephrase, or leave it to the explanation just above the `dat$Gender` code block.

Reply:
Would "when we subset the data frame by a single column" be a better alternative?

Reply:
Or "When we extract a single column from a data frame"?

Reply:
The `extract`-variant is better IMHO.

`class(subset(dat, select = BloodPressure))` on the other hand would still be a `data.frame` and thus wouldn't fit to the 2nd part of your sentence.